### PR TITLE
Check if the app is the latest release or not. (0.24.0)

### PR DESCRIPTION
**Pull request by nikhilmaske-2001**
**Bumped version: 0.24.0**

Currently this PR is completely ready

**Screenshot:**
When update is available
image
When update is not available
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -210,6 +213,28 @@ async function Main() {
...
+
+        // Storing data in form of JSON
+        var data = await response.json();
+        var latestVersion = data[0].tag_name;
+        var latestVersionLink = `https://github.com/vasani-arpit/WBOT/releases/tag/${latestVersion}`;
+        var myVersion = 'v' + require('../package.json').version;
+        
+        spinner.stop("Checking for an Update... Completed");
+
+        if(latestVersion !== myVersion) {
```
Comment (vasani-arpit) [inline src/index.js:232]: Let's say user is ahead of the `latestVersion`, in that case this condition will be true and user will see the message which is a wrong behavior.

**Comment by vasani-arpit** [comment]
We follow [semantic versioning](https://semver.org/). 

I don't think you can compare it as a string. I suggest you to use https://www.npmjs.com/package/semver for it.

**Comment by nikhilmaske-2001** [comment]
@vasani-arpit Updated the PR, I have used `lt()` function to compare `myVersion` and `latestVersion`.

File: src/index.js [hunk 2]
```diff
@@ -210,6 +213,28 @@ async function Main() {
...
+        var latestVersion = data[0].tag_name;
+        var latestVersionLink = `https://github.com/vasani-arpit/WBOT/releases/tag/${latestVersion}`;
+        var myVersion = 'v' + require('../package.json').version;
+
+        spinner.stop("Checking for an Update... Completed");
+
+        if(lt(myVersion, latestVersion)) {
+            console.log(`An Update is available for you.\nPlease download the latest version ${latestVersion} of WBOT from ${latestVersionLink}`);
+        } else {
+            console.log("There is no update available.")
```
Comment (vasani-arpit) [inline src/index.js:235]: No need to log anything if there is no update.

Comment (nikhilmaske-2001) [reply src/index.js:235]: Updated the PR #281 , Please take a look at #281. (I am messed up with this branch on my local repo that's why I have created a new PR #281 for this same task.)

**Comment by nikhilmaske-2001** [comment]
Closing this PR as I have created new PR #281

**Pull request closed without merging**